frontend/src/components/common/input/input.tsx

```diff
@@ -25,17 +29,22 @@ const Input: FC<Props> = ({
   name,
   placeholder = '',
   type = 'text',
+  titleClass = '',
+  labelClass = '',
+  inputClass = '',
 }) => {
   const { field } = useFormControl({ name, control });
 
   return (
-    <label className={styles.label}>
-      <span className={styles.title}>{label}</span>
+    <label className={getValidClasses(styles.label, labelClass)}>
+      <span className={getValidClasses(titleClass ?? styles.title)}>
+        {label}
+      </span>
       <input
         {...field}
         type={type}
         placeholder={placeholder}
-        className={styles.input}
+        className={getValidClasses(inputClass ?? styles.input)}
```

Comment:
```suggestion
        className={getValidClasses(styles.input, inputClass)}
```
you need to add classes